system_test/helpers/aest_api.erl

```diff
@@ -0,0 +1,420 @@
+-module(aest_api).
```

Comment:
I like this API module. We should refactor the other suites to use it as well (and add to it).